doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,245 @@
 
 ## Table of Contents
 * [5.1 Introduction](#5.1)
-* [5.2 Heading](#5.2)
-* [5.3 Heading](#5.3)
+* [5.2 Installer requirements](#5.2)
+* [5.3 Descriptor file definition](#5.3)
 
 <a name="5.1"></a>
 ## 5.1 Introduction
 
+**must**: Requirements that are marked as _must_ are considered mandatory and must exist in the reference implementation and implemented by installer. The same applies to _must not_.
+
+**should**: Requirements that are marked as _should_ are expected to be fulfilled by the reference implementation but it is up to each installer to accept an implementation targeting this reference implementation that is not reflecting on any of those requirements. The same applies to _should not_.
+
+**may**: Requirements that are marked as _may_ are considered optional. The same applies to _may not_.
+
 <a name="5.2"></a>
-## 5.2 Heading
+## 5.2 Installer requirements
+<a name="5.2.1"></a>
+### 5.2.1 General
+The Descriptor File defines the unique configuration required by installer in a common schema. 
+It would specialize the installer type per user's implementation requirements.
+It would be validated at the very beginning of the deployment.
+It's installer's responsibility to translate the descriptor file to adapt with its own configuration. 
+Thanks to the descriptor file, the NFVi infrastructure deployment could be completed in one step run.
 
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.gen.ins.01` | Installer | Installer **should** accept a descriptor file to finish deployment.|
```

Comment:
@ChilionChan Should we keep this requirement as a **should**? Without the support of the descriptor file, any installer cannot comply with the structure that we have listed down. It is possible that the installer doesn't have support for this yet as mentioned in `req.gen.ins.03` but that is just a gap in the implementation. For installer to be used that gap must be filled first. 
Opinion?

Reply:
I update it to must.